### DISCUSSION TITLE
Unsupported Nvim releases failing

### DISCUSSION THREAD

**Discussion by Aprilskysky** [neovim/neovim#31725] [Q&A]
This repo provides best-effort, [unsupported builds](https://github.com/neovim/neovim-releases/releases) (as opposed to the [supported builds](https://github.com/neovim/neovim/releases)) of Nvim for some legacy systems or other quirky scenarios:

Linux (built with glibc 2.17, unlike the [supported builds](https://github.com/neovim/neovim/releases) which use a newer glibc)

Unsupported Nvim releases failing

**Comment by clason** [comment]
They're unsupported for a reason.
